expression/bench_test.go

```diff
@@ -265,6 +265,29 @@ func (g *defaultGener) gen() interface{} {
 	return nil
 }
 
+// charInt64Gener is used to generate int which is equal to char's ascii
+type charInt64Gener struct{}
+
+func (g *charInt64Gener) gen() interface{} {
+	rand := time.Now().Nanosecond()
+	rand = rand % 128
```

Comment:
We may not need to limit it under 128, see https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_char
> CHAR() arguments larger than 255 are converted into multiple result bytes

I think we can limit it under 1024